### PR TITLE
Small multiserver tutorial fixes from DOC-913

### PR DESCRIPTION
**Pull request by ernopp**
Previously approved here https://github.com/spatialos/UnrealGDK/pull/848

### CODE REVIEW

File: SpatialGDK/Documentation/content/get-started/tutorial.md [hunk 1]
```diff
@@ -76,14 +76,14 @@ In this section you’ll run a [local deployment](https://docs.improbable.io/ref
 
 1. In a text editor, open `UnrealGDKThirdPersonShooter\spatial\default_launch.json` and in the `layer_configurations` section set `rows` to `2`. This instructs SpatialOS that you will be launching two server-workers.
```
Comment (m-samiec) [inline SpatialGDK/Documentation/content/get-started/tutorial.md]: The GDK default is to use the generated launch config, so changing this variable should be done via the GDK settings. I'd suggest removing this line, and I'll add one after setting the number of servers.

Comment (m-samiec) [inline SpatialGDK/Documentation/content/get-started/tutorial.md]:
```suggestion
```

File: SpatialGDK/Documentation/content/get-started/tutorial.md [hunk 2]
```diff
@@ -76,14 +76,14 @@ In this section you’ll run a [local deployment](https://docs.improbable.io/ref
...
-5. In the Unreal Editor, in the SpatialOS GDK toolbar, select **Start** (the green play icon). This opens a terminal window and runs the [`spatial local launch`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-local-launch#spatial-local-launch) command, which starts the [SpatialOS Runtime](https://docs.improbable.io/reference/latest/shared/glossary#the-runtime).
-6. It's ready when you see `SpatialOS ready. Access the inspector at http://localhost:21000/inspector`.
-7. From the Unreal Editor toolbar, select **Play** to run the game. This starts two SpatialOS server-worker instances and two SpatialOS client-worker instances locally, in your Unreal Editor.
+1. In the Unreal Editor, in the SpatialOS GDK toolbar, select **Start** (the green play icon). This opens a terminal window and runs the [`spatial local launch`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-local-launch#spatial-local-launch) command, which starts the [SpatialOS Runtime](https://docs.improbable.io/reference/latest/shared/glossary#the-runtime).
```
Comment (m-samiec) [inline SpatialGDK/Documentation/content/get-started/tutorial.md]:
```suggestion
1. Within the **Play** drop-down menu open **SpatialOS Settings**.
1. Change **Launch configuration file description > Workers > 0 > Rectangle grid row count** to **2**.
1. In the Unreal Editor, in the SpatialOS GDK toolbar, select **Start** (the green play icon). This opens a terminal window and runs the [`spatial local launch`](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-local-launch#spatial-local-launch) command, which starts the [SpatialOS Runtime](https://docs.improbable.io/reference/latest/shared/glossary#the-runtime).
```

**Comment by ernopp** [comment]
Discussed with Mike that even though this new flow for changing number of workers in the settings pane is not optimal, we will not be updating it before 1.0

**Pull request merged**